tools/wptserve/wptserve/server.py

```diff
@@ -285,8 +285,12 @@ def finish_handling(self, request, response, handler):
             try:
                 handler(request, response)
             except HTTPException as e:
+                self.logger.error("HTTPException in handler: %s" % e)
```

Comment:
I think I'd like us to try only logging stacks for 500-type errors (here or below), and at warning level. For other exceptions (e.g. 404) the normal logging should be sufficient.

Reply:
I've updated the PR to focus on logging only when the handler throws a 5xx 
code. LMKWYT!